File: Comparisons/loops.c
#include <stdio.h>

int main() {
  int i;

  for (i = 1; i < 11; ++i)          //for loop
  {
    printf("%d ", i);
  }

  int j = 5;                        //while loop
  while(j >= 0) {
    printf("%d ", j);
    j--;
  }

int num=1;	
	do	{                            //do-while loop 
		printf("%d\n",2*num);
		num++;		
	}
    while(num<=10);
	

  return 0;
}

File: Comparisons/loops.rb
#while loop
x = gets.chomp.to_i

while x >= 0
    puts x
    x = x -1
end

#until loop
x = gets.chomp.to_i

until x < 0
    puts x
    x -= 1
end

#for loop
x = gets.chomp.to_i

for i in 1..x do
    puts x - i
end